fibsem/detection/detection.py

```diff
@@ -1,45 +1,115 @@
 #!/usr/bin/env python3
 import logging
 from dataclasses import dataclass
-
+from enum import Enum
 import numpy as np
 
 from fibsem.microscope import FibsemMicroscope
 from scipy.spatial import distance
 from skimage import feature
 
 from fibsem import conversions
-from fibsem.detection.utils import Feature, FeatureType
+# from fibsem.detection.utils import Feature, FeatureType
 from fibsem.imaging import masks
 from fibsem.segmentation.model import SegmentationModel
 from fibsem.structures import BeamType, MicroscopeSettings, Point
 
-FEATURE_COLOURS_UINT8 = {
-    FeatureType.ImageCentre: (255, 255, 255),
-    FeatureType.LamellaCentre: (255, 0, 0),
-    FeatureType.LamellaLeftEdge: (255, 0, 0),
-    FeatureType.LamellaRightEdge: (255, 0, 0),
-    FeatureType.NeedleTip: (0, 255, 0),
-    FeatureType.LandingPost: (255, 255, 255),
-}
+_DETECTION_V3 = True
+if _DETECTION_V3:
+
+
+    from abc import ABC, abstractmethod
+    # TODO: maybe static methods? probs nicer to use?
+    # e.g. needle_tip = NeedleTip.detect(img, mask, point)
+    # can store color info there too?
+
+    @dataclass
+    class Feature_v3(ABC):
```

Comment:
rename this from Feature_v3 to Feature as we are just removing the old one,

Reply:
done